content/millennium/r4/encounters/encounter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency, etc. A patient may have one medical record number with multiple encounter numbers per facility or organization. There is substantial variance between organizations in the definition of an encounter and what events are aggregated together to constitute an encounter.
+
+## Patch
+
+Patch an existing encounter.
+
+    PATCH /Encounter/:id
+
+_Implementation Notes_
+
+* Only operations on the paths listed below are supported.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {'Content-Type': 'application/json-patch+json', 'If-Match': 'W/"&lt;Current version of the Encounter resource>"'} %>
+<%= headers fhir_json: true %>
+
+### Patch Operations
+
+<%= patch_definition_table(:encounter_patch, :r4) %>
+
+### Example
+
+#### Request
+
+    PATCH https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Encounter/1621910
+
+#### Body
+
+<%= json(:r4_encounter_patch) %>
+
+#### Response
+
+<%= headers status: 200 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Tue, 26 Mar 2019 15:42:29 GMT
+Etag: W/"10"
+Last-Modified: Tue, 26 Mar 2019 15:42:27 GMT
+Server-Response-Time: 2260.237021
+Status: 200 OK
+Vary: Origin
+X-Request-Id: 47306a14c8a2c3afd4ab85aa9594101d
+X-Runtime: 2.260092
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
```

Comment:
Add the `Errors` section.

Reply:
Added. https://github.com/cerner/fhir.cerner.com/pull/240/commits/2f088fe9f198e6cb9d48c1be55d513f5eb8c9f72#diff-97d84edc87e609cdbf80d6ad20cbc681R20

Reply:
Make sure that the links to `errors` and `OperationOutcomes` is present before merging.